static/js/redux/ui/advising.jsx

```diff
@@ -92,6 +104,29 @@ class Advising extends React.Component {
     }
   }
 
+  addRemoveAdvisor(advisor, added) {
```

Comment:
why is it saying you added this? isn't this on cs310-spring21 already?

Reply:
No this is from Jeanie's branch.

Reply:
ahh yes i thought it was merged